src/compose.js

```diff
@@ -0,0 +1,3 @@
+import flowRight from 'lodash.flowright';
```

Comment:
Do we need these 2 files? I mean it's gonna look like "API surface" to me. We can just require these whenever we want. and I'm not seeing a day we wouldn't use lodash, you know, since it's awesome :grin: